src/common/country-base.ts

```diff
@@ -134,7 +134,7 @@ export const african_countries = [
 ]
 
 // special partnership payments for this countries
-export const cpa_plan_countries = ['za', 'ec']
+export const cpa_plan_countries = ['za', 'ec', 'br', 'lk', 'uy']
```

Comment:
I think Switzerland is missing @NikitK-deriv

Reply:
Switzerland is in european union :)